types/styled-components/index.d.ts

```diff
@@ -21,6 +22,7 @@ declare global {
 
 import * as CSS from "csstype";
 import * as React from "react";
+import hoistNonReactStatics = require('hoist-non-react-statics');
```

Comment:
Prefer `import ... from` to be consistent with previous imports.

Reply:
Actually I'd prefer to change the other imports to `import = require`. `react` is not an ES module, and as long as `esModuleInterop` doesn't default to `true` for `.d.ts` parsing it's better to be explicit about the commonjs import.

Reply:
I'd agree but that is a) part of another change and b) inconsistent with virtually everybody else. That boat has long sailed where we can insist to not import react as an esmodule. `import * as React` is close enough. `import React = require()` is very uncommon syntax.